sig-windows/annual-report-2024.md

```diff
@@ -4,40 +4,29 @@
 
 1. What work did the SIG do this year that should be highlighted?
 
-<!--
-   Some example items that might be worth highlighting:
-   - Major KEP advancement
-   - Important initiatives that aren't tracked via KEPs
-   - Paying down significant tech debt
-   - Governance and leadership changes
--->
+  - Continued working on [Image pull per runtime class](https://github.com/kubernetes/enhancements/issues/4216)
+  - [Windows support for memory pressure eviction](https://github.com/kubernetes/kubernetes/pull/122922)
+  - Migrated CI jobs to community infra
+  - Unit and e2e testing improvements
+  - [Documentation](https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes/) improvements for adding Windows nodes
 
 2. Are there any areas and/or subprojects that your group needs help with (e.g. fewer than 2 active OWNERS)?
 
+   No
 
 3. Did you have community-wide updates in 2024 (e.g. KubeCon talks)?
 
-<!--
-  Examples include links to email, slides, or recordings.
--->
+  - [SIG Windows Retrospective and Windows Image Building Deep Dive, Kubecon EU](https://sched.co/1YhiZ)
+  - [What's new with SIG Windows, Kubecon NA](https://sched.co/1hoxQ)
 
 4. KEP work in 2024 (v1.30, v1.31, v1.32):
-<!--
-   TODO: Uncomment the following auto-generated list of KEPs, once reviewed & updated for correction.
 
-   Note: This list is generated from the KEP metadata in kubernetes/enhancements repository.
-      If you find any discrepancy in the generated list here, please check the KEP metadata.
-      Please raise an issue in kubernetes/community, if the KEP metadata is correct but the generated list is incorrect.
--->
-
-<!-- 
   - Alpha
     - [4802 - Windows Graceful Node Shutdown](https://github.com/kubernetes/enhancements/tree/master/keps/sig-windows/4802-windows-node-shutdown) - v1.32
     - [4885 - Windows CPU and Memory Affinity](https://github.com/kubernetes/enhancements/tree/master/keps/sig-windows/4885-windows-cpu-and-memory-affinity) - v1.32
 
   - Beta
     - [2258 - Node log query](https://github.com/kubernetes/enhancements/tree/master/keps/sig-windows/2258-node-log-query) - v1.30
```

Comment:
should this mention DSR and overlay https://github.com/kubernetes/enhancements/issues/5100?

Reply:
That happened in 2025 :P